.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--
+
+Thank you for contributing to the Firebase community! Please fill out the pull request form below
+and make note of the following:
+
+Run the linter and test suite
+==============================
+Run `npm test` to make sure your changes compile properly and the tests all pass on your local machine. 
+We've hooked up this repo with continuous integration to double check those things for you. 
+
+Add tests (if applicable)
+==============================
+Most non-trivial changes should include some extra test coverage. If you aren't sure how to add
+tests, feel free to submit regardless and ask us for some advice.
+
+Sign our CLA
+==============================
+Please sign our Contributor License Agreement (https://cla.developers.google.com/about/google-individual)
+before sending PRs. We cannot accept code without this.
+
+Add an reviewer
+==============================
+Please add @laurenzlong as a reviewer.
```

Comment:
This probably shouldn't be in our template. We can do what other orgs do and have a bot autoassign this. Could make for a good Cloud Functions integration #meta